docs/sources/monitor/monitor-windows.md

```diff
@@ -0,0 +1,299 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tutorials/scenarios/monitor-windows/
+description: Learn how to use Grafana Alloy to monitor Windows servers and desktops
+menuTitle: Monitor Windows
+title: Monitor Microsoft Windows servers and desktops with Grafana Alloy
+weight: 250
+---
+
+# Monitor Microsoft Windows servers and desktops with {{% param "FULL_PRODUCT_NAME" %}}
```

Comment:
Shouldn't we be instructing users to use the Windows tile in the connections page?

<img width="672" alt="Screenshot 2025-03-26 at 10 11 24" src="https://github.com/user-attachments/assets/761f6ce0-ead1-4f60-8fe2-81be219741d1" />

There's more than one way of doing the same thing? Not sure what is recommended?

Reply:
Hi @ptodev, the purpose of this repo was to help Open Source users achieve their goals with Alloy and provide them usable syntax examples. This example using the wizard is great for Grafana Cloud users using the wizard and I belive will eventually go into a learning journey: https://grafana.com/docs/learning-journeys/

Reply:
I'm just worried that this work feels like it's independent from existing docs and examples. It'd be good if it integrates with them so that the information isn't scattered all around. I didn't know about the learning journeys, they look cool but it's yet another place that has duplicate information. It's also going to be difficult to keep this up to date if it doesn't live in the same repo as the one developers work in.